ts/private/configuration.bzl

```diff
@@ -0,0 +1,43 @@
+"""A terminal rule collecting verbosity and worker support information"""
+
+ConfigurationInfo = provider(
```

Comment:
I know this mirrors "config_setting", but naming this thing "configuration" feels too close to "ts_config" to me. Will be confusing in communication. Can we name it something else? "Options" maybe?

Reply:
renamed it to options